pages/linux/toolbox-create.md

```diff
@@ -0,0 +1,20 @@
+# toolbox create
+
+> Create a new `toolbox` container.
+> More information: <https://manned.org/toolbox-create.1>.
+
+- Create a `toolbox` container for a different distribution other than the host:
+
+`toolbox create --distro {{distribution}}`
+
+- Create a `toolbox` container for a different distribution release other than the host:
+
+`toolbox create --release {{release}}`
+
+- Change the name of the image used to create the `toolbox` container:
+
+`toolbox create --image {{name}}`
```

Comment:
I assume this command doesn't work on its own. We should always provide minimal working examples. As far as I understood you have several default images for each release of each distro. So which image name does this command edit?

Reply:
Sure I will add an example. Usually, there is only 1 default image for each release of each distro (sometimes there might be testing images, etc). 
![image](https://user-images.githubusercontent.com/26346867/182434663-44f5dbc2-fe5e-4d0d-904e-dc2d9e4bd9e5.png)